config/config.json

```diff
@@ -9,5 +9,8 @@
     	"maxInterval": 15,
     	"minInterval": 10,
     	"intervalStep": 5
-    }
+    },
+    "parameters": {
+        "template_name": ethtemplate
```

Comment:
why is ethtemplate not withing double quotes? my intelij detected some syntex error

Reply:
gcp does not accept special characters in the template name. It is not a syntax error, don't worry ;)

Reply:
Actually this does not fulfill JSON structure and it is the origin of the main.py error you had this afternoon. I checked that when fixing this main.py script worked.

Reply:
Okay but we don't need to call this script from main.py as karim explained above.